src/js/view/controls/components/volumetooltip.js

```diff
@@ -7,25 +7,38 @@ export default class VolumeTooltip extends Tooltip {
     constructor(_model, name, ariaText, svgIcons) {
         super(name, ariaText, true, svgIcons);
 
+        const localization = _model.get('localization');
+
         this._model = _model;
         this.volumeSlider = new Slider('jw-slider-volume jw-volume-tip', 'vertical');
         this.volumeSlider.setup();
-        this.volumeSlider.element().classList.remove('jw-background-color');
 
-        setAttribute(this.volumeSlider.element(), 'aria-label', this._model.get('localization').volumeSlider);
+        const volumeSliderElement = this.volumeSlider.element();
+        volumeSliderElement.classList.remove('jw-background-color');
+
+        setAttribute(volumeSliderElement, 'aria-label', localization.volumeSlider);
+        setAttribute(this.container, 'tabindex', '0');
 
-        this.addContent(this.volumeSlider.element());
+        this.addContent(volumeSliderElement);
 
         this.volumeSlider.on('update', function (evt) {
             this.trigger('update', evt);
             this.el.focus();
         }, this);
 
+        this.container.addEventListener('blur', () => {
```

Comment:
these event listeners should be removed, at the very least when the volumeTooltip is destroyed.

Reply:
this has not been resolved. I think it's up to the person commenting to resolve a comment

Reply:
actually, no need to use an arrow function to call a function
```
this.container.addEventListener('blur', this.closeTooltip);
this.container.addEventListener('focus', this. openTooltip);
.
.
.
destroy() {
        this.container.removeEventListener('blur', this.closeTooltip);
        this.container.removeEventListener('focus', this.openTooltip);
    }
```

Reply:
or do you....